vault/changelog.release.2021-12-28.md

```diff
@@ -0,0 +1,92 @@
+---
+id: N1Mn5p6O2tVVkdDM0Ys1w
+title: '0.75'
+desc: ''
+updated: 1640719141612
+created: 1640715852170
+---
+
+Dendron 0.75 has sprouted  🌱
+
+This is the last release of the year!
+
+When deleting notes, did you just break some links elsewhere? Now Dendron makes sure to give you a heads up, about any links that will break, with the latest update to the `Dendron: Delete Node` command.
+
+Speaking of broken links, Dendron has an entirely new command: **Convert Link**! Have links you need to update, or links you'd like to otherwise transform into other formats (like usertags or hashtags)? Take a look at [[Convert Link|dendron://dendron.dendron-site/dendron.topic.refactoring#convert-link]] and give it a shot.
+
+By default, backlinks are listed in alphabetical order. Now you choose to sort those backlinks by last update timestamp (`updated` attribute in note frontmatter). Learn more ([[here|dendron://dendron.dendron-site/dendron.topic.workbench#backlink-view-command-palette]]).
+
+### Highlights
+
+- enhance(commands): `Dendron: Delete Node` command warns about links that will break, prompting users before deletion ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#delete-node]])
+- feat(refactor): New `Dendron: Convert Link` command 
+- enhance(views): adding optional _last update_ sort ordering for backlinks view
+
+### Everything Else
+
+- enhance(pods): better support for GitHub Issue and task notes ([[docs|dendron://dendron.dendron-site/dendron.topic.pod.builtin.github-issue.publish#aliasmapping]])
+- enhance(lookup): better error message when using schema lookup
+- fix(markdown): lag in the editor when there's a x-vault link to a non-existent vault
+- fix(markdown): correctly offset frontmatter line count in doctor preview for `findBrokenLinks`
+- fix(publish): excluding the domain of a published hierarchy will cause publishing to throw an error
+- fix(publish): hamburger display in wrong position on safari
+- fix(lookup): autocomplete will sometimes not initialize
+- fix(views): looping behavior when performing rename when note graph is open
+- fix(commands): seed commands broken by internal refactoring
+
+### Community
+
+#### General Updates
+
+- **New User Tuesdays**: Starting on **January 11th**, Dendron will be having regular sessions for new users to ask questions and provide feedback on Dendron. Take a look at the [Dendron Event Calendar](https://link.dendron.so/luma) for these and other events!
```

Comment:
don't need to change but in the future, we don't need to track this via switchy since registration is already a signal